snapcraft/tests/test_commands_init.py

```diff
@@ -37,6 +37,16 @@ def test_init_must_write_snapcraft_yaml(self):
         fake_logger = fixtures.FakeLogger(level=logging.INFO)
         self.useFixture(fake_logger)
 
+        expected_yaml = """name: # the name of the snap
+version: # the version of the snap
+summary: # 79 char long summary
+description: # a longer description for the snap
+confinement: devmode # devmode means the snap doesn't support confinement"""
```

Comment:
Suggested wording:

`confinement: devmode # replace by "strict" to enforce system access only via declared interfaces`

This gives a hint about both what it does and how to move forward.